src/datastore/postgres/schema/v5.ts

```diff
@@ -0,0 +1,26 @@
+import { IDatabase } from "pg-promise";
+
+// tslint:disable-next-line: no-any
+export async function runSchema(db: IDatabase<any>) {
+    // Create schema
+    await db.none(`
+    CREATE TABLE metrics_users (
+        user_id TEXT NOT NULL,
+        remote BOOLEAN,
```

Comment:
and also this, probably can infer remote data from the users table also

Reply:
I removed the metrics_users table.